internal/nfs/controller/controllerserver.go

```diff
@@ -75,7 +74,7 @@ func (cs *Server) CreateVolume(
 	req *csi.CreateVolumeRequest) (*csi.CreateVolumeResponse, error) {
 	res, err := cs.backendServer.CreateVolume(ctx, req)
 	if err != nil {
-		return res, fmt.Errorf("failed to create CephFS volume: %w", err)
+		return res, status.Errorf(status.Code(err), "failed to create CephFS volume: %v", err)
```

Comment:
just return err from here?

Reply:
sure! The same is already done on `DeleteVolume`, so it makes sense.